sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -470,6 +472,13 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class ProcessingContext
+    {
+        internal ProcessingContext() { }
+        public int AttemptNumber { get { throw null; } }
+        public System.Exception? LastException { get { throw null; } }
```

Comment:
Is this the last exception from the service from a retry?  Or anywhere in the library?  Would it make sense for it to be RequestFailedException?

Reply:
This would be an exception that occurred as part of the pipeline - so most likely something like an IOException, SocketException, or an exception thrown from another policy. RFEs are created higher in the stack. We could wrap it in an RFE here, but I don't see the benefit.